tests/Team.py

```diff
@@ -105,6 +105,20 @@ def testMembers(self):
         self.assertRaises(AssertionError, self.team.add_membership, user, "admin")
         self.team.remove_membership(user)
 
+    def testTeamMembership(self):
+        user = self.g.get_user("jacquev6")
+        self.assertListKeyEqual(self.team.get_members(), None, [])
+        self.assertFalse(self.team.has_in_members(user))
+        self.team.add_to_members(user)
```

Comment:
add_to_members() API call is deprecated. Use `add_membership` instead.

Reply:
changed both in test and ReplayData